server/app/views/LoginForm.java

```diff
@@ -98,7 +98,8 @@ public Content render(Http.Request request, Messages messages, Optional<String>
         div().withClasses("fixed", "w-screen", "h-screen", "bg-gray-200").with(content));
 
```

Comment:
A small mistake from earlier: the dev tools aren't appearing on the login form. This is irrelevant when the bypass flag is on, but came up when testing Seattle staging before the flag was flipped. This file will be deleted soon.